keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -887,6 +919,16 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+Same as above: all the components must have support for it to work.
+Specifically, kube-apiserver will filter this field if it doesn't recognize/support it.
+If the kubelet doesn't support the field, it will not pull the image, and the path will not be present for the volume manager
```

Comment:
I don't follow the phrase "and the path will not be present for the volume manager" in this context but it makes sense after reading the next paragraph re old kubelet with missing volume plugin, presumably extending the capability to allow oci volume source mounting by the runtime.  Maybe reorder or clarify?

Reply:
Yep, done